payments/integration-test/step_definitions/support/steps.js

```diff
@@ -0,0 +1,131 @@
+
+let responseToCheck;
+let serviceToDelete;
+let orgToDelete;
+
+Given('Payments running', async function () {
+    const response = await healthCheck();
+    assert.strictEqual(response.status, 200);
+});
+Given('GPS running', async function () {
+    const response = await gpsHealthCheck();
+    assert.strictEqual(response.status, 200);
+});
+Given('GPD running', async function () {
+    const response = await gpdHealthCheck();
+    assert.strictEqual(response.status, 200);
+});
+Given('IUV Generator running', async function () {
+	//TODO expose health check on APIM
+    //const response = await iuvGenHealthCheck();
+    //assert.strictEqual(response.status, 200);
+});
+Given('DonationService running', async function () {
+	//TODO expose health check on APIM
+    //const response = await donationHealthCheck();
+    //assert.strictEqual(response.status, 200);
+});
+Given('ApiConfig running', async function () {
+    const response = await apiConfigHealthCheck();
+    assert.strictEqual(response.status, 200);
+});
+
+Given('the service {string} for donations', async function (serviceId) {
+    serviceToDelete = serviceId;
+    await deleteService(serviceToDelete)
+    let response = await createService({
+        "id": serviceId,
+        "name": "DonationpagoPAservice",
+        "description": "DonationpagoPAservice",
+        "transferCategory": "tassonomia-1",
+        "status": "ENABLED",
+        "endpoint": donation_host,
+        "basePath": "/donations/paymentoptions",
+        "properties": [
+            {
+                "name": "amount",
+                "type": "NUMBER",
+                "required": true
+            },
+            {
+                "name": "description",
+                "type": "STRING"
+            }
+        ]
+    })
+    assert.strictEqual(response.status, 201);
+});
+Given('the creditor institution {string} enrolled to donation service {string}', async function (orgId, serviceId) {
+    orgToDelete = orgId
+    await deleteOrganization(orgToDelete)
+    let response = await createOrganization(orgId, {
+        "companyName": "Comune di Milano",
+        "enrollments": [
+            {
+                "serviceId": serviceId,
+                "iban": "IT00000000000000001",
+                "officeName": "Ufficio Tributi",
+                "segregationCode": "77",
+                "remittanceInformation": "causale di pagamento"
+            }
+        ]
+    })
+    
+    assert.strictEqual(response.status, 201);
+});
+When('the client sends the DemandPaymentNoticeRequest', async function () {
+    responseToCheck = await demandPaymentNotice(`<soapenv:Envelope xmlns:pafn="http://pagopa-api.pagopa.gov.it/pa/paForNode.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/">
+        <soapenv:Body>
+            <pafn:paDemandPaymentNoticeRequest>
+                <idPA>77777777777</idPA>
```

Comment:
it doesn't sound good to hard coded here info like `idPA`, `idBrokerPA` , `idStation` and `idServizio`.
I think it is more correct as step parameters

Reply:
I added idPa  as parameter. For now I would leave the others unchanged